TradeItIosEmsApi/TradeItAuthLinkResult.h

```diff
@@ -11,8 +11,8 @@
 
 @interface TradeItAuthLinkResult : TradeItResult
 
-@property (nullable) NSString *userId;
-@property (nullable) NSString *userToken;
+@property (nonnull) NSString *userId;
+@property (nonnull) NSString *userToken;
```

Comment:
I say leave them nullable.  The only way they would be null is a severe server bug, however, the app still shouldn't crash in that scenario, and the app can't prevent the server from returning null values in JSON.